Traceback (most recent call last):
  File "R0-multi-step.py", line 59, in <module>
    plt.plot(R0_average)
  File "/apps/developers/libraries/python-libs/3.1.0/1/python-3.6.5/lib/python3.6/site-packages/matplotlib-2.0.0-py3.6-linux-x86_64.egg/matplotlib/pyplot.py", line 3307, in plot
    ax = gca()
  File "/apps/developers/libraries/python-libs/3.1.0/1/python-3.6.5/lib/python3.6/site-packages/matplotlib-2.0.0-py3.6-linux-x86_64.egg/matplotlib/pyplot.py", line 950, in gca
    return gcf().gca(**kwargs)
  File "/apps/developers/libraries/python-libs/3.1.0/1/python-3.6.5/lib/python3.6/site-packages/matplotlib-2.0.0-py3.6-linux-x86_64.egg/matplotlib/pyplot.py", line 586, in gcf
    return figure()
  File "/apps/developers/libraries/python-libs/3.1.0/1/python-3.6.5/lib/python3.6/site-packages/matplotlib-2.0.0-py3.6-linux-x86_64.egg/matplotlib/pyplot.py", line 535, in figure
    **kwargs)
  File "/apps/developers/libraries/python-libs/3.1.0/1/python-3.6.5/lib/python3.6/site-packages/matplotlib-2.0.0-py3.6-linux-x86_64.egg/matplotlib/backends/backend_tkagg.py", line 81, in new_figure_manager
    return new_figure_manager_given_figure(num, figure)
  File "/apps/developers/libraries/python-libs/3.1.0/1/python-3.6.5/lib/python3.6/site-packages/matplotlib-2.0.0-py3.6-linux-x86_64.egg/matplotlib/backends/backend_tkagg.py", line 89, in new_figure_manager_given_figure
    window = Tk.Tk()
  File "/apps/developers/compilers/python/3.6.5/1/default/lib/python3.6/tkinter/__init__.py", line 2020, in __init__
    self.tk = _tkinter.create(screenName, baseName, className, interactive, wantobjects, useTk, sync, use)
_tkinter.TclError: no display name and no $DISPLAY environment variable
